e2e-tests/functions

```diff
@@ -622,3 +623,28 @@ kubectl_bin() {
     rm "$LAST_OUT" "$LAST_ERR"
     return ${exit_status}
 }
+
+patch_secret() {
+    local secret=$1
+    local key=$2
+    local value=$3
+    
+    kubectl patch secret $secret -p="{\"data\":{\"$key\": \"$value\"}}"
+}
+
+getSecretData() {
+    local secretName=$1
+    local dataKey=$2
+    local  data=`kubectl get secrets/${secretName} --template={{.data.${dataKey}}} | base64 -D`
+    echo "$data"
+}
+
+check_mongo_auth() {
+    local uri="$1"
+
+    ping=$(run_mongo "db.runCommand({ ping: 1 }).ok" "$uri" "mongodb" "" "--quiet" | egrep -v 'I NETWORK|W NETWORK|Error saving history file|Percona Server for MongoDB|connecting to:|Unable to reach primary for set|Implicit session:|versions do not match')
+    desc "ping return"
+    if [ ${ping} != 1 ]; then
```

Comment:
```suggestion
    if [ "${ping}" != "1" ]; then
```
the overwise syntax error if `ping` is an empty string

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/387/commits/02ea61fc44d3bb8158b9c31ed03106d0f44182fb